home-manager/common.nix

```diff
@@ -34,7 +34,7 @@
   home = {
     sessionVariables = {
       # https://unix.stackexchange.com/questions/4859/visual-vs-editor-what-s-the-difference
-      EDITOR = "vim";
+      EDITOR = "micro"; # If you forgot the keybind: https://github.com/zyedidia/micro/blob/c15abea64c20066fc0b4c328dfabd3e6ba3253a0/runtime/help/defaultkeys.md
```

Comment:
walk respect this, but the keybinds are very vimsh

micro: ctrl+q
walk: q - https://github.com/antonmedv/walk/issues/105


Hmm.. 🤔